<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelWidget</class>
 <widget class="QWidget" name="ChannelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>67</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>67</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>67</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="currentChannel">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>50</y>
     <width>118</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,100);
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;私人兆赫&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="channel" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#channel {
background-color: rgba(0,0,0,100);
border: 0px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="maskLeft" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#maskLeft {background-color: rgba(0,0,0,100);}</string>
   </property>
   <widget class="QLabel" name="leftArrow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>11</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>11</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>11</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="maskRight" native="true">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>0</y>
     <width>117</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>117</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>117</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#maskRight {background-color: rgba(0,0,0,100);}</string>
   </property>
   <widget class="QLabel" name="rightArrow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>11</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>11</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>11</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>channel</zorder>
  <zorder>currentChannel</zorder>
  <zorder>maskLeft</zorder>
  <zorder>maskRight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
